ops/testing.py

```diff
@@ -1253,14 +1261,17 @@ def storage_get(self, storage_name_id, attribute):
                 'ERROR invalid value "{}/{}" for option -s: storage not found'.format(name, index))
 
     def storage_add(self, name: str, count: int = 1):
+        if name.startswith('/'):
+            raise model.ModelError('test storage name cannot start with "/"')
+
         if name not in self._storage_list:
             self._storage_list[name] = {}
         result = []
         for i in range(count):
-            index = self._storage_id_counter
-            self._storage_id_counter += 1
+            index = self._storage_index_counter
+            self._storage_index_counter += 1
             self._storage_list[name][index] = {
-                "location": "/{}/{}".format(name, index)
+                "location": os.path.join(self._harness_tmp_dir.name, name, str(index)),
```

Comment:
If `_harness_tmp_dir: Path`: 
`self._harness_tmp_dir / name / str(index)`

Reply:
harness_tmp_dir is a tempfile.TemporaryDirectory.  It's not a string or a pathlib path.